File: src/main/java/com/soprasteria/osca/domain/referentiel/rubriques/sousrubriques/SousRubriques.java
package com.soprasteria.osca.domain.referentiel.rubriques.sousrubriques;

import com.soprasteria.osca.domain.referentiel.rubriques.sousrubriques.dev.FeuilleDeRouteDev;
import com.soprasteria.osca.domain.referentiel.rubriques.sousrubriques.fonc.FeuilleDeRouteFonc;
import com.soprasteria.osca.domain.referentiel.rubriques.sousrubriques.ux.FeuilleDeRouteUX;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class SousRubriques {

    private String titre;
    private String keySousRubriques;
    private List<FeuilleDeRouteUX> sousRubriqueUx;
    private List<FeuilleDeRouteFonc> sousRubriqueFonctionnelle;
    private List<FeuilleDeRouteDev> sousRubriqueDeveloppement;
}

File: src/main/webapp/osca/common/domain/Referentiel/Rubriques/SousRubriques/SousRubriques.ts
import { FeuilleDeRouteFonc } from './Fonc/FeuilleDeRouteFonc';
import { FeuilleDeRouteDev } from './Dev/FeuilleDeRouteDev';
import { FeuilleDeRouteUX } from './UX/FeuilleDeRouteUX';

export interface SousRubriques {
    titre: string;
    keySousRubriques: string;
    sousRubriqueUx: FeuilleDeRouteUX[];
    sousRubriqueDeveloppement: FeuilleDeRouteDev[];
    sousRubriqueFonctionnelle: FeuilleDeRouteFonc[];
}
